src/main/java/net/mcreator/ui/action/AcceleratorMap.java

```diff
@@ -36,6 +36,10 @@ public AcceleratorMap(ActionRegistry actionRegistry) {
 				KeyStroke.getKeyStroke(KeyEvent.VK_F, Toolkit.getDefaultToolkit().getMenuShortcutKeyMaskEx()));
 		actionKeyStrokeMap.put(actionRegistry.showReplaceBar,
 				KeyStroke.getKeyStroke(KeyEvent.VK_R, Toolkit.getDefaultToolkit().getMenuShortcutKeyMaskEx()));
+		actionKeyStrokeMap.put(actionRegistry.openFile,

```

Comment:
F6 and Shift+F6 are used for move and rename in other IDEs so this may be confusing, I don't think we need a shortcut for this

Reply:
I'll fix that tomorrow (busy with states editor now).